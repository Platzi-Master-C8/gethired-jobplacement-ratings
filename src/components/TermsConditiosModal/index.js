import TermsConditiosModal from './TermsConditiosModal';

export { TermsConditiosModal };
