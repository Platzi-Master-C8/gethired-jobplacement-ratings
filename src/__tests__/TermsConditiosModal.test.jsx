import React from 'react';

import { render } from '@testing-library/react';
import { TermsConditiosModal } from 'Components/TermsConditiosModal';

describe('Component <TermsConditiosModal/>', () => {
    test('renders TermsConditiosModal', () => {
        const { asFragment } = render(<TermsConditiosModal />);
        expect(asFragment()).toMatchSnapshot();
    });
});
