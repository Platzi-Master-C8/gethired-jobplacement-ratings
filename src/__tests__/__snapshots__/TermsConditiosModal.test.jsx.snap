// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <TermsConditiosModal/> renders TermsConditiosModal 1`] = `
<DocumentFragment>
  <div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12 css-oav0mg-MuiGrid-root"
    >
      <a
        class="MuiTypography-root MuiTypography-caption MuiLink-root MuiLink-underlineNone css-v181wt-MuiTypography-root-MuiLink-root"
      >
        Terminos y Condiciones
      </a>
    </div>
  </div>
</DocumentFragment>
`;
